lesson_10_task_01.py

```diff
@@ -0,0 +1,35 @@
+from selene import config
+from rozetka_home_page import RozetkaHomePage
+from rozetka_product_page import RozetkaProductPage
+from rozetka_search_results_page import RozetkaSearchResultsPage
+
+
+def test_rozetka():
+    config.timeout = 40
+
+    rozetka_home_page = RozetkaHomePage()
+    rozetka_search_results_page = RozetkaSearchResultsPage()
+    rozetka_product_page = RozetkaProductPage()

```

Comment:
do not create objects before you need them
also only one page object should be created in a test